src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -142,7 +142,7 @@ private[scoverage] abstract class ScoverageSensorInternal extends Sensor {
         settings
           .get(ScoverageReportPathPropertyKey)
           .toOption
-          .getOrElse(defaultScoverageReportPath)
+          .getOrElse(defaultScoverageReportPath.toString)
```

Comment:
Well, since you already made the effort of using `Paths` instead of `Strings` , why don't we use it for the **report path** too?